packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -500,9 +536,8 @@ export class RunningSummarizer implements IDisposable {
 
     private async generateSummaryWithLogging(message: string, safe: boolean): Promise<GenerateSummaryData | undefined> {
         const summarizingEvent = PerformanceEvent.start(this.logger, {
-            eventName: "Summarizing",
+            eventName: "SummarizerRun",
```

Comment:
@arinwt, @vladsud, @anthony-murphy - heads up on this rename.

Reply:
I think we should reconsider these names. (correct me if I'm wrong though)

My understanding is the new full event name will be: "... :Summarizer:Run:SummarizerRun".
Currently, when we "run" the summarizer, it means we begin the heuristics and start the RunningSummarizer. But here "run" means a specific summary is being created/tracked.  It seems like maybe we want to reconsider?

Reply:
Bah, good point about `Summarizer.run`. I was trying to get something unique to describe each iterative turn through the RunningSummarizer that generates a Summary.

How about................... `:Summarizer:Running:GenerateSummary_start`, and the property can be called `summaryGenTag`? Or maybe `summarizerRunTag` still works? Other ideas?

I want some other keyword (i.e. not just `summaryTag`) so we're not depending on distinguishing between Summarizing/Summarizer/Summary - I've found for myself and others that those words all run together in the brain.

Reply:
That seems fine to me. I think there used to be another perf event within ContainerRuntime with a similar name, but since that was removed I agree that `GenerateSummary_*` is probably a good name.

Personally I like `summaryTag`, but yeah `summaryGenTag` is probably fine too.

Reply:
i like matching the method name in cases like this, as it makes look up in the code easy and obvious

Reply:
Thanks for the feedback 👍